e2e_tests/.flake8

```diff
@@ -4,9 +4,57 @@ max-line-length = 100
 # We ignore F401 in __init__.py because it is expected for there to be
 # "unused imports" when defining a "regular" package. (This file is
 # implicitly executed when the package is imported, and the imports would
-# be used by the importer.) We ignore patch_saver_restore.py because it includes
-# a near-verbatim TensorFlow function with a small patch.
-per-file-ignores = __init__.py:F401 patch_saver_restore.py:E111,E114,
+# be used by the importer.)
+per-file-ignores =
```

Comment:
Do you think this is worth a ticket and/or TODO referencing that ticket?

Reply:
it was on the ml sys agenda to discuss the plan for addressing these. I'd like to have this discussion before we can add concrete guidance here.